doc/whats_new.rst

```diff
@@ -12,6 +12,56 @@ Version 0.18
 Changelog
 ---------
 
+.. _model_selection_changes:
+
+Model Selection Enhancements and API Changes
+--------------------------------------------
+
+  - **The ``model_selection`` module**
+
+    The classes and functions from the formerly :mod:`sklearn.cross_validation`,
+    :mod:`sklearn.grid_search` and :mod:`sklearn.learning_curve` are
+    grouped together into a single module available at
+    :mod:`sklearn.model_selection`. There are some major changes to these
+    newly moved classes and functions. Refer below to know about the
+    changes.
+
+  - **Data-independent CV splitters enabling nested cross-validation**
+
+    The new cross-validation splitters, defined in the
+    :mod:`sklearn.model_selection`, are not initialized with any
+    data-dependent parameters. Instead they expose a :func:`split` method
```

Comment:
I'd make it clearer that they used to be, e.g. "are no longer initialized with data-dependent parameters such as `y`, allowing their use in a nested setting."